_posts/2022-04-05-maliit-release-2.2.1.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: Maliit 2.2.2 Release
+date: 2022-04-05 19:02
+categories: release news
+---
+
+## Maliit framework
+
+- Re-show the keyboard on Wayland surrounding text changes
+
+## Maliit keyboard
+
+- Remove the legacy CC-BY 3.0 licensed artwork.
+- Simplify ActionsToolbar by removing need for states
+- Visual spacing improvements for Keyboard and WordRibbon
+- Add a visible separator between word suggestions
+- Select a minimum width for word suggestion labels
+- Make word suggestion font size proportional to WordRibbon height
+- Hide WordRibbon when in cursor mode
+- Make wordRibbon height proportional to Keyboard keys
+- Only load emoji keyboard QML when state is EMOJI
+- Fix typo in internal API (@sunweaver)
+- Adjust glib deprecation fix to work with glib < 2.68
+- Update Italian translation
+- Ensure all icon names are corrrect and using symbolic variants
+- Fix some typos in QML imports
+- Make sursor mode available for Japanese plugin
+- Enable use of anthy-unicode instead of anthy
+
+## Where can I get it?
+
+* framework:
+  [maliit-framework-2.2.1.tar.gz](https://github.com/maliit/framework/archive/refs/tags/2.2.1.tar.gz)
+* keyboard:
+  [maliit-keyboard-2.2.1.tar.gz](https://github.com/maliit/keyboard/archive/refs/tags/2.2.1.tar.gz)
+
+## What is it?
+
+Maliit provides a flexible and cross-platform input method framework. It has a
+plugin-based client-server architecture where applications act as clients and
+communicate with the Maliit server via input context plugins. Maliit is an open
+source framework (LGPL 2.1 or later) and an open source on-screen keyboard (LGPL 3.0 only).
+
+Maliit is being used by [KDE for Plasma](https://blog.martin-graesslin.com/blog/2021/03/using-maliit-keyboard-in-a-plasma-wayland-session/),
+[LG for webOS](https://github.com/webosose/maliit-framework-webos),
```

Comment:
Does anybody know if Steam Deck's on-screen keyboard is Maliit-based?

Reply:
I'm pretty sure the virtual keyboard in Game Scope on the deck is an internal thing and not based on Maliit.

Reply:
Yes, steam deck uses the keyboard that comes with steam client itself and not maliit.